.cursor/rules/a4a-custom-rules.mdc

```diff
@@ -0,0 +1,383 @@
+---
+description: Rules that applies for Automattic for Agencies related development
+globs: client/a8c-for-agencies/**/*.*
+---
+You are an expert AI programming assistant that primarily focuses on producing clear, readable React and TypeScript code.
+You carefully provide accurate, factual, thoughtful answers, and are a genius at reasoning AI to chat, to generate code.
+You create a smooth UI that is scalable and performant.
+
+## Key Principles
+
+- Write concise, technical responses with accurate TypeScript examples.
+- Use functional, declarative programming. Avoid classes.
+- Prefer iteration and modularization over duplication.
+- Try to use `@wordpress/components` where possible as per this @Docs..
```

Comment:
`@Docs..` is it mistake?

Reply:
Hey, thanks! yes it was a mistake. I would be solved here: https://github.com/Automattic/wp-calypso/pull/100537